src/views/Search.vue

```diff
@@ -16,17 +16,17 @@
 
             <form action="" class="searchBar rf-col-11 my-4">
                 <div class="input-group mb-3">
-                <input type="search" class="form-control border-bottom-green formresearch p-4" v-model="recherche" placeholder="Rechercher un mot clé, une expression, une référence..." aria-describedby="basic-addon2">
+                <input type="search" class="form-control border-bottom-green formresearch p-4" v-model="newResearch" @keydown.enter.prevent="Research(newResearch)" placeholder="Rechercher un mot clé, une expression, une référence..." aria-describedby="basic-addon2">
```

Comment:
Même remarque que précédemment : il vaut mieux se binder sur l'événement `submit` à l'échelle du formulaire.

Par exemple, actuellement, si j'essaye de soumettre le formulaire au clavier en déplaçant le focus sur le bouton et en appuyant sur <entrée>, et bien cela ne fonctionne pas :)

Enfin, pour des raisons d'a11y, il faut ajouter un label à cet input, et le cacher via la classe `.sr-only`.

```suggestion
                <input type="search" class="form-control border-bottom-green formresearch p-4" v-model="newResearch" placeholder="Rechercher un mot clé, une expression, une référence…">
```